js/index.js

```diff
@@ -38,5 +38,76 @@ const siteContent = {
 };
 
 // Example: Update the img src for the logo
-let logo = document.getElementById("logo-img");
+const logo = document.getElementById("logo-img");
 logo.setAttribute('src', siteContent["nav"]["img-src"])
+
+// update the nav items
+const nav = document.querySelectorAll('nav a');
+nav.forEach((a, i) => {
+  a.textContent = siteContent.nav[`nav-item-${i+1}`];
+  a.style.color = 'green';
+});
+
+// add two more items to the nav
+const firstAnchor = document.createElement('a');
+const lastAnchor = document.createElement('a');
+firstAnchor.textContent = 'First';
+lastAnchor.textContent = 'Last';
+firstAnchor.style.color = 'green';
+lastAnchor.style.color = 'green';
+document.querySelector('header nav').appendChild(lastAnchor);
+document.querySelector('header nav').prepend(firstAnchor);
+
+// updates the cta
+document.querySelector('button').textContent = siteContent.cta["button"];
+document.querySelector('.cta-text h1').innerHTML = 'DOM<br> Is<br> Awesome';
```

Comment:
This is fine, but you should have used the object provided to update your DOM. 
Could you find a way to insert those tags without rewriting the string?